db/db.js

```diff
@@ -4,35 +4,31 @@
 const uuidv4 = require("uuid/v4");
 const Knex = require("knex");
 const knexConfig = require("./knexfile");
-const { Model } = require("objection");
-
-const EmailHash = require("./models/emailhash");
 
 const HIBP = require("../hibp");
 const getSha1 = require("../sha1-utils");
 
 const knex = Knex(knexConfig);
-Model.knex(knex);
 
 
-const DBUtils = {
-  async addUnverifiedEmailHash(email) {
-    return await EmailHash.query().insert(
-      {email: email, sha1: getSha1(email), verification_token: uuidv4(), verified: false}
-    );
+const DB = {
+  async addSubscriberUnverifiedEmailHash(email) {
+    const res = await knex("subscribers").insert(
+      { email: email, sha1: getSha1(email), verification_token: uuidv4(), verified: false }
+    ).returning("verification_token");
+    return res[0];
```

Comment:
Note: `knex` queries return arrays, so have to return the `0`th item.

Reply:
Suggestion: while we're refactoring, let's proactively add a stub function for encryption (that just returns the email as-is for now) and use it everywhere. Later when we add encryption, we can just add it in one place and not have to refactor everything again.

Reply:
Can we do in a follow-up? I'm picturing this as part of moving `sha1-utils.js` into a new `crypto.js` module which will pull the private key in only when needed.